docs/client/basic/sections/templates.md

```diff
@@ -210,27 +210,26 @@ event object and template instance. See the [Event Maps section](#eventmaps)
 for details.
 <!-- TODO Update the link to full docs for Event Maps -->
 
-{{> autoApiBox "Template#rendered"}}
+{{> autoApiBox "Template#onRendered"}}
 
-The function assigned to this property is called once for every instance of
+The functions added with this method are called once for every instance of
 *Template.myTemplate* when it is inserted into the page for the first time.
 
-This _rendered_ callback can be used to integrate external libraries that aren't
-familiar with Meteor's automatic view rendering, and need to be initialized
-every time HTML is inserted into the page. Use the
-[`created`](#template_created) and
-[`destroyed`](#template_destroyed) callbacks to perform
-initialization or clean-up on any objects.
+These _rendered_ callbacks can be used to integrate external libraries that
+aren't familiar with Meteor's automatic view rendering, and need to be
+initialized every time HTML is inserted into the page. Use the callbacks added
```

Comment:
I think we can just say "You can also use `onCreated` and `onDestroyed` to perform initialization or clean-up..."